tests/unit/parsers/test.manifestjson.js

```diff
@@ -1730,6 +1730,31 @@ describe('ManifestJSONParser', () => {
         expect(warnings.length).toEqual(0);
       });
 
+      it('adds a warning if the icon is SVG but the dimensions of the image are not the same', async () => {
```

Comment:
Nitpick: let's make this test description a bit shorter e.g. `it('does add a warning if the icon is a non square svg', ...)`

Reply:
done.
Thank for the feedback